ground/src/main/res/layout/fragment_terms_service.xml

```diff
@@ -56,7 +56,7 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_margin="8dp"
-        android:text="@{viewModel.termsOfServiceText}"
+        android:text="@{viewModel.getTermsOfServiceText()}"
```

Comment:
Why not expose this as a LiveData<String> field `termsOfServiceText`? I think it would look something like:

`termsOfServiceText = liveData { emit(termsOfServiceRepository.getTermsOfService()?.text) }`

Reply:
This is cool! I didn't know we could do this.